SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -141,4 +141,4 @@ The `spdlog` documentation states that it is supported on the following platform
 
 ### Vulnerability Disclosure Policy [7.i]
 
-There is no Vulnerability Disclosure Policy for `spdlog`.
+The Vulnerability Disclosure Policy it's defined in the vendored Quality Declaration.
```

Comment:
Let's point to our own policy, for issues that affect ROS 2.

Reply:
```suggestion
The Vulnerability Disclosure Policy is defined in the vendored Quality Declaration.
```